modules/salt/pages/formula-yomi.adoc

```diff
@@ -0,0 +1,153 @@
+* In the [guimenu]``Hostname`` field, enter the hostname for the client to use.
+Leave this blank if you are using DHCP to provide the hostname.
+* In the [guimenu]``Machine Id`` field, enter a machine identification number for the client.
+Leave this blank to have systemd generate one automatically.
+* In the [guimenu]``Target`` field, enter a systemd target unit.
+. Click btn:[Save Formula] to save your configuration.
+. Apply the highstate.
+
+
+
+.Procedure: Configuring the Yomi Storage Formula
+. Navigate to the menu:Formulas[Yomi Storage] tab, and set these parameters in the menu:Partitions[Config] section:
+* In the [guimenu]``Labels`` field, select the default partition table type to use.
+* In the [guimenu]``Initial Gap`` field, select the default amount of space to leave before each partition.
+For example: [systemitem]``1{nbsp}MB``, or use [systemitem]``0`` to leave no space between partitions.
+. For each device that you want to create, in the menu:Partitions[Devices] section, click btn:[+], and set these parameters:
+* In the [guimenu]``Device`` field, type the mount point for the device.
+For example, [systemitem]``/dev/sda``.
+* In the [guimenu]``Label`` field, select the partition table type to use, if it is different from the default label you selected.
+* In the [guimenu]``Initial Gap`` field, select the amount of space to leave before each partition, if it is different from the default space you specified.
+. For each partition that you want to create, in the menu:Partitions[Devices > Partitions] section, click btn:[+], and set these parameters:
+* In the [guimenu]``Partition Number`` field, enter a number for the partition.
+The number you enter here is be appended to the device name to identify the partition.
+For example, partition number [systemitem]``1`` on device [systemitem]``/dev/sda`` can be identified as [systemitem]``/dev/sda1``.
+* In the [guimenu]``Partition Name`` field, enter a name for the partition.
+Leave this blank if you have entered a partition number in the previous field.
+* In the [guimenu]``Partition Size`` field, enter a size for the partition.
+For example: [systemitem]``500{nbsp}MB``.
+Use [systemitem]``rest`` to use all the remaining free space.
+. For each file system that you want to create, in the [guimenu]``Filesystems`` section, click btn:[+], and set these parameters:
+* In the [guimenu]``Partition`` field, select the partition to create the file system on.
+For example, [systemitem]``/dev/sda1``.
+* In the [guimenu]``Filesystem`` field, select the filesystem type to create.
+* In the [guimenu]``Mountpoint`` field, type the mount point for the filesystem.
+For example: [systemitem]``/`` for root.
+. Click btn:[Save Formula] to save your configuration.
+. Apply the highstate.
+
+
+[[NOTE]]
+====
+If you want to use LVM or RAID on your devices, click btn:[+] in the appropriate sections, and complete the details for your environment.
+====
```

Comment:
maybe reference the documentation, there are some tricks here, like for example that the RAID can be repartitioned.

Reply:
We probably need to document it properly here, but i was being lazy.